src/Common.targets

```diff
@@ -12,12 +12,5 @@
 		<!-- Enable ShouldWriteErrorOnInvalidXaml for all the sample and test projects of this solution -->
 		<ShouldWriteErrorOnInvalidXaml>True</ShouldWriteErrorOnInvalidXaml>
```

Comment:
I guess this is more suitable in `Directory.Build.props`?

Reply:
Yes. It's not a useful file anymore, it used to be present before Directory.Build.* existed.